cmd/nodecmd/local.go

```diff
@@ -240,43 +286,60 @@ func localStartNode(_ *cobra.Command, args []string) error {
 		avalanchegoBinaryPath,
 		numNodes,
 		nodeConfig,
-		anrSettings,
+		connectionSettings,
+		nodeSettings,
 		avaGoVersionSetting,
 		network,
 	)
 }
 
-func localStopNode(_ *cobra.Command, _ []string) error {
-	return node.StopLocalNode(app)
+func localStopNode(_ *cobra.Command, args []string) error {
+	if len(args) == 1 {
```

Comment:
local stop node cmd has maxargs 1. Why not make it exact 1, so we always have 1 arg